Before: fd97ed68d5a27a885760e2f7db43e4d31889c44a392d74b3f0a9aa33c5c97703
After: 33bfbc981a0607f4ba3019729ce32d3a0171520c560873cc5a84d42f841bc3e9
Patch: 4fd12c58f967c093fcdc2b826a2ef94fccc60cf33e283267451b28b618453aeb
--- corvid-features/plugin.rb	2012-08-09 13:34:29.694263314 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1,6 +0,0 @@
-install {
-  empty_directory 'resources'
-  empty_directory 'resources/latest'
-  copy_file       'resources/.gitignore'
-  install_feature 'test_spec', run_bundle: true, say_if_installed: false
-}
--- resources/.gitignore	2012-08-09 13:34:14.000844233 +1000
+++ /dev/null	2012-07-30 09:37:23.066384672 +1000
@@ -1 +0,0 @@
-latest
